src/com/lftechnology/realworldassign/ClothesFashion.java

```diff
@@ -0,0 +1,99 @@
+package com.lftechnology.realworldassign;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class manipulates different clothes of the provided fashion.
+ * 
+ * @author bibhushan<bibhushanjoshi@lftechnology.com>
+ * 
+ */
+public class ClothesFashion extends Product implements ProductManagementLayout {
```

Comment:
Do model class needs to implements the service, 
you would better design a abstract model class Product and then extends Product class by ClothesFashion and related classes.
Donot implement service interfaces (ProductManagementLayout) for Model class(Product). Rather create new service class ClothesService that would implement ProductManagement service and use model (ClotherFashion) inside ClothesService class